src/main.rs

```diff
@@ -103,6 +106,16 @@ fn settings() -> io::Result<Settings> {
                     DEFAULT_THRESHOLD.code_str(),
                 )),
         )
+        .arg(
+            Arg::with_name(KEEP_OPTION)
+                .value_name("KEEP")
+                .short("k")
+                .long(KEEP_OPTION)
+                .multiple(true)
```

Comment:
Not sure if you'll like this structure, but I went with a multiple argument (with one value each) so the usage is:

`--keep regex1 --keep regex2`

I thought this might be preferable to having a single very convoluted regex for more complex setups.

Reply:
Seems reasonable.